wallet/src/Ergvein/Wallet/Util.hs

```diff
@@ -79,4 +89,6 @@ currencyToCurrencyCode c = case c of
 currencyCodeToCurrency :: CurrencyCode -> ETC.Currency
 currencyCodeToCurrency c = case c of
   BTC -> ETC.BTC
+  TBTC -> ETC.BTC
   ERGO -> ETC.ERGO
+  TERGO -> ETC.ERGO
```

Comment:
Partial function, what if indexer send code for DASH?

Reply:
Added error message